RunGcm-filters-leticia.ipynb

```diff
@@ -0,0 +1,3088 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    filter_cpu = gcm_filters.Filter(grid_vars={'area': area, 'wet_mask': wet_mask}, **specs)
</pre><p>While it does not really matter how you name the filter, I suggest to delete the "_cpu" here. I just used this in my example notebook to make clear that we are filtering on CPUs, not on GPUs. But this may not be important for your notebook because we are more focusing on the science here. :)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/oceanhackweek/ohw22-proj-gcmfilters/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>